docs/axes/cartesian/time.md

```diff
@@ -23,20 +23,23 @@ When providing data for the time scale, Chart.js supports all of the formats tha
 
 ## Configuration Options
 
-The following options are provided by the time scale. They are all located in the `time` sub options. These options extend the [common tick configuration](README.md#tick-configuration).
+The following options are provided by the time scale. You may also set options provided by the [common tick configuration](README.md#tick-configuration).
 
 | Name | Type | Default | Description
 | -----| ---- | --------| -----------
-| `displayFormats` | `Object` | | Sets how different time units are displayed. [more...](#display-formats)
-| `isoWeekday` | `Boolean` | `false` | If true and the unit is set to 'week', iso weekdays will be used.
-| `max` | [Time](#date-formats) | | If defined, this will override the data maximum
-| `min` | [Time](#date-formats) | | If defined, this will override the data minimum
-| `parser` | `String` or `Function` | | Custom parser for dates. [more...](#parser)
-| `round` | `String` | `false` | If defined, dates will be rounded to the start of this unit. See [Time Units](#time-units) below for the allowed units.
-| `tooltipFormat` | `String` | | The moment js format string to use for the tooltip.
-| `unit` | `String` | `false` | If defined, will force the unit to be a certain type. See [Time Units](#time-units) section below for details.
-| `stepSize` | `Number` | `1` | The number of units between grid lines.
-| `minUnit` | `String` | `'millisecond'` | The minimum display format to be used for a time unit.
+| `scales.distribution` | `String` | `linear` | How data is plotted. Valid options are `linear` and `series`. [more...](#scale-distribution)
```

Comment:
I would remove `Valid options are linear and series.` to keep the table shorter: it's detailed in the `more` link.